kitsune/sumo/jinja2/includes/common_macros.html

```diff
@@ -143,9 +143,9 @@ <h4 class="mzp-c-menu-item-title">{{ _('Mozilla Account') }}</h4>
             <ul class="mzp-c-menu-item-list sumo-nav--sublist">
               <li><a href="{{ url('wiki.document', 'access-mozilla-services-firefox-account' )}}">{{ _('What is it?') }}</a></li>
               <li><a href="{{ url('products.product', 'mozilla-account')}}">{{ _('Get Help') }}</a></li>
-              <li><a href="{{ url('wiki.document', 'switching-devices') }}">{{ _('Sync My Data') }}</a></li>
-              <li></li>
               <li><a href="https://accounts.firefox.com/reset_password">{{ _('Reset Password') }}</a></li>
+              <li></li>
```

Comment:
why do we have this empty item? Can we drop it?

Reply:
The "Get Help" has to be alone on the right, and leaving that item empty let it do so without changing any other code.